torch/fx/proxy.py

```diff
@@ -10,6 +10,9 @@
 class TracerBase:
     graph: Graph
 
+    def output(self, result : Argument) -> Node:
```

Comment:
It was intentional that there only be a `create_node` function in the tracer. So this function should probably not exist.